pkgs/top-level/all-packages.nix

```diff
@@ -5603,11 +5597,11 @@ with pkgs;
   gcc = gcc6;
   gcc-unwrapped = gcc.cc;
 
-  gccStdenv = if (!stdenv.isDarwin) then stdenv else stdenv.override {
+  gccStdenv = if stdenv.cc.isGNU then stdenv else stdenv.override {
     allowedRequisites = null;
     cc = gcc;
-    # Include unwrapped binaries like AS, etc. and remove libcxx/libcxxabi
-    extraBuildInputs = [ stdenv.cc.cc ];
+    # Remove libcxx/libcxxabi
+    extraBuildInputs = [ ];
```

Comment:
This breaks `tests.cc-wrapper-gcc` on darwin, but there's probably a nicer way to fix it.

Reply:
I did the same method, but made it darwin-specific and specifically clang.